indexer/services/roundtable/src/helpers/pnl-ticks-helper.ts

```diff
@@ -166,6 +160,30 @@ export async function getPnlTicksCreateObjects(
   return newTicksToCreate;
 }
 
+/**
+ * Gets a list of subaccounts that were last updated more than 0.9 *
+ * PNL_TICK_UPDATE_INTERVAL_MS ago.
+ *
+ * @param mostRecentPnlTicks
+ * @param blockTime
+ */
+export function getAccountsToUpdate(
```

Comment:
how about instead of that we get the list of subaccounts that haven't been updated this hour? For example(all pseudocode):
```
const blockTime = new Date(blockTime);
const normalizedBlockTime = normalizeStartTime(new Date(blockTime)); // 12:00:01 -> 12:00:00
const lastUpdatedBlockTime = accountToLastUpdatedBlockTime[accountId];
const normalizedLastUpdatedBlockTime = normalizeStartTime(lastUpdatedBlockTime);

// process if normalizedBlockTime != normalizedLastUpdatedBlockTime

Reply:
that's cleaner. done.